docs/cel2/faq.md

```diff
@@ -64,7 +68,7 @@ Dango diverged from Alfajores L1 at block [24940100](https://celo-alfajores.bloc
 
 ### I noticed you used the same chainID for Alfajores and Dango. Isn't that dangerous?
 
-Good catch! Yes, we're using the same chain ID for both Alfajores and Dango. This means that transactions from one testnet can be executed on the other testnet as well. But as both networks are testnets, this is not a problem.
+Good catch! Yes, we're using the same chain ID for both Alfajores and Dango. This means that transactions from one testnet can be executed on the other testnet as well. But as both networks are testnets, this is not a problem. Furthermore, Dango is set to go away in fall 2024.
```

Comment:
will it not be turned off now. Then I would say, it has been shut down with the launch of Alfajores 2024.

Reply:
It hasn't yet been shut down, we're keeping it for at least another week - does the wording work in that context?